### DISCUSSION TITLE
Cant map n key

### DISCUSSION THREAD

**Discussion by H4ckint0sh** [neovim/neovim#30233] [Q&A]
I am trying to map n key to center the screen while looping search results by following mapping:

`vim.keymap.set("n", "n", "nzzzv")
`

But it does not work and when I run `map n` command it says 'no mapping found'.  But if I map the same as below it is working fine:

`vim.keymap.set("n", "<leader>n", "nzzzv")
`

while :

`vim.keymap.set("n", "N", "Nzzzv")
`

is working as intended and also is shown in `map N` command.

**Comment by gabrielfrimodig** [comment]
Try use `noremap` and `silent` options with `vim.keymap.set` like: 
```lua
vim.keymap.set("n", "n", "nzzzv", { noremap = true, silent = true })
```
